src/logger/sentry.ts

```diff
@@ -31,7 +31,7 @@ export const defaultOptions = {
 export async function initSentry() {
   try {
     const dist = `${VersionNumber.buildVersion}`; // MUST BE A STRING
-    const release = `${VersionNumber.appVersion} (${dist})`; // MUST BE A STRING
+    const release = `${VersionNumber.bundleIdentifier}@${VersionNumber.appVersion}+${dist}`; // MUST BE A STRING
```

Comment:
`bundleIdentifier` maps to the respective identifiers for iOS and Android: https://github.com/APSL/react-native-version-number